spec/jobs/scheduler/scheduler_spec.rb

```diff
@@ -7,57 +7,79 @@
 describe "scheduler" do
   let(:release) { Bosh::Template::Test::ReleaseDir.new(File.join(File.dirname(__FILE__), "../../..")) }
   let(:job) { release.job("scheduler") }
-  let(:template) { job.template("config/application.properties") }
+  let(:template) { job.template("config/scheduler.yml") }
   let(:properties) { YAML.safe_load(fixture("scheduler.yml").read) }
-  let(:rendered_template) { template.render(properties) }
+  let(:rendered_template) { YAML.safe_load(template.render(properties)) }
 
-  context "config/application.properties" do
-    it "does not set username nor password if not configured" do
+  context "config/scheduler.yml" do
+    it "does set neither username nor password if not configured" do
       properties["autoscaler"]["scheduler"] = {
         "health" => {
-          "port" => 1234
+          "port" => 1234,
+          "unprotected_endpoints" => []
         }
       }
-      rendered_template = template.render(properties)
 
-      expect(rendered_template).to include("scheduler.healthserver.port=1234")
-      expect(rendered_template).to include("scheduler.healthserver.basicAuthEnabled=false")
-      expect(rendered_template).to include("scheduler.healthserver.username=")
-      expect(rendered_template).to include("scheduler.healthserver.password=")
+      rendered_template = YAML.safe_load(template.render(properties)) # klappt nicht
```

Comment:
```suggestion
      rendered_template = YAML.safe_load(template.render(properties))
```
Let's remove this comment.